src/avatar/styled-components.js

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import {styled} from '../styles';
+import type {StylePropsT} from './types';
+
+function getSize(props: StylePropsT) {
+  const {$size, $theme} = props;
+
+  const defaultSize = $theme.sizing.scale1000;
+  const size = $size || defaultSize;
```

Comment:
no need for `|| defaultSize` here

Reply:
flow complains about accessing `undefined` on the object (`$theme.sizing[undefined]`)

Reply:
@nadiia resolving for now, but let me know if you have strong feeling about this